scripts/globals/mobskills/august_melee_axe.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------
+--  August Melee - Axe
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/mobskills")
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 1
+    local accmod = 1
+    local dmgmod = 1
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
+    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.BLUNT, info.hitslanded)
+    target:takeDamage(dmg, mob, xi.attackType.PHYSICAL, xi.damageType.BLUNT)
+    skill:setMsg(xi.msg.basic.HIT_DMG)
```

Comment:
the set msg actually shouldn't be needed on melee substitute skills, the setMobSkillAttack system normally handles that automatically

Reply:
Doesn't handle it automatically unless I have that line:
<img width="203" alt="image" src="https://user-images.githubusercontent.com/1389729/228510427-0b151ad8-a4c9-4578-8066-7ece9a3ee444.png">

Reply:
now wondering if I broke that when I refactored the flags

Reply:
Appears I did. lets leave the setMsg and I'll fix the other mobs skill attack scripts later.